gazebo_plugins/scripts/wheel_slip_drawbar_pull_publisher.py

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/env python3
+#
+# Copyright 2022 Open Source Robotics Foundation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# Desc: helper script for publishing drawbar pull wrenches
+# Author: Steve Peters
+#
+import argparse
+import math
+import rclpy
+import sys
+from geometry_msgs.msg import Wrench
+from rclpy.node import Node
+
+kDefaultTopic = '/drawbar_pull'
+kDefaultForceIncrement = 10
+kDefaultMaxForce = 70
+kDefaultPublishInterval = 2.0
+
+class WheelSlipDrawbarPullPublisher(Node):
+    def __init__(self, args):
+        super().__init__('wheel_slip_drawbar_pull_publisher')
+        parser = argparse.ArgumentParser(
+            description='Publish drag forces as a wrench representing drawbar pull on the back of a vehicle.')
+        parser.add_argument('-t', '--topic', type=str, default=kDefaultTopic,
+                            help='The topic name for publishing a drawbar pull wrench')
```

Comment:
I generally recommend relying on ROS remaps for configuring topic names, simply to have one way to do it. For example, hard code the topic below when creating the publisher and then at runtime you can remap the node with:

```
./wheel_slip_drawbar_pull_publisher.py --ros-args -r drawbar_pull:=new_topic_name
```

Or from a launch file with a `<remap>` tag.

Though I don't think it hurts to also provide this option. So I'll leave it up to you if you want to keep it or not.

Reply:
good idea; I've hard-coded the topic name in https://github.com/ros-simulation/gazebo_ros_pkgs/pull/1364/commits/dcca4afd7c4876d96efd84e1c117a926f973b242 and will rely on remapping to change it